Add support for iOS 13 SceneDelegate (#21)
braintree/popup-bridge-ios: README.md

        [POPPopupBridge setReturnURLScheme:@"com.my-app.popupbridge"];
        return YES;
    }
    ```
    

>     1. Inspect the redirect URL and then call `PopupBridge:openURL` from either your app delegate or your scene delegate.

Reviewer: ```suggestion
    1. Inspect the return URL and then call `PopupBridge:openURL` from either your app delegate or your scene delegate.
```

Minor suggestion for consistency.

>      If you're using the `SceneDelegate` class introduced in iOS 13, call `PopupBridge:openURL` from within the  `scene:openURLContexts` delegate method. Pass the URL on the appropriate `UIOpenURLContext`. 

Reviewer: ```suggestion
     If you're using `UISceneDelegate` (introduced in iOS 13), call `PopupBridge:openURL` from within the  `scene:openURLContexts` delegate method. Pass the URL on the appropriate `UIOpenURLContext`. 
```

     
     ```objectivec
     - (void)scene:(UIScene *)scene openURLContexts:(NSSet<UIOpenURLContext *> *)URLContexts {
        for (UIOpenURLContext *urlContext in URLContexts) {
            NSURL *url = [urlContext URL];
            if ([url.scheme localizedCaseInsensitiveCompare:@"com.my-app.popupbridge"] == NSOrderedSame) {
            [POPPopupBridge openURL:urlContext.URL];
            }
        }
    }
    ```
    

>     If you aren't using `SceneDelegate`, call `PopupBridge:openURL` from within the  `application:openURL:` delegate method of your app delegate.

Reviewer: ```suggestion
    If you aren't using `UISceneDelegate`, call `PopupBridge:openURL` from within the  `application:openURL:` delegate method of your app delegate.
```
